docs: final company report

ginge270 opened the pull request:
# Description

This adds the final company report for Thoth Tech T1 2022.

## Type of change

- [ ] Documentation (update or new)

# How Has This Been Tested?

N/A

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have requested a review from @tancnle @talzid @kats8  on the Pull Request

ginge270 committed:
docs: final company report

Commit:
docs: run prettier

This file needs a lot of work before it is functional. However, it is an exact replicate of the submission.

ghost commented:
Fails vale due to headings.
File will require a lot of work before functional.

tancnle commented:
The table format is also broken 😢

tancnle committed:
docs: convert report to GitHub Flavoured markdown

This commit ensure the original docx is converted to the GitHub
Flavoured markdown format using `pandoc`.

tancnle commented:
@ginge270 @talzid I have attempted to convert the original docx to the correct GitHub Flavoured Markdown format using [`pandoc`](https://pandoc.org/) (view the rendered report [here](https://github.com/thoth-tech/handbook/blob/2e374d134e0aa57b081301adb3d977594c672cf3/docs/company/reports/t1-2022/company-report-v3-final.md)). 

There are still some small issues to address but the layout looks generally better.

For reference, this is the command I ran to convert:

```shell
pandoc -f docx -t gfm -o foo.md report-v3.docx
```

Commit:
docs: removed duplicate and simplified TOC

tancnle committed:
docs: standardise deliverables headers

Only use header level 2 to 4 in the TOC.

tancnle committed:
fix: broken showcase links

tancnle committed:
fix: remove extraneous spacing cause by <p> tags

tancnle committed:
fix: remove empty lines and fix lists

tancnle committed:
fix: remove column width styles

tancnle committed:
docs: convert to PNG for better web rendering

tancnle committed:
docs: add forecast images

tancnle committed:
fix: link to handbook repo
